src/components/ExpandedViewMode/ExpandedViewMode.js

```diff
@@ -28,18 +35,19 @@ function ExpandedViewMode({ goalAddress, goal, updateGoal, onClose, squirrels })
         />
         <div className='expanded_view_main'>
           <ExpandedViewModeContent
+            squirrels={squirrels}
             goalAddress={goalAddress}
             updateGoal={updateGoal}
             goal={goal}
-            squirrels={squirrels}
           />
           <RightMenu
+            squirrels={squirrels}
```

Comment:
I think I messed this up during merge... I think it should be removed (my fault)
```suggestion
```